servicemesh-install/topics/tutorial-kiali.adoc

```diff
@@ -37,61 +37,101 @@ tracing                tracing-istio-system.127.0.0.1.nip.io
 
 +
 . Launch a browser and navigate to https://${KIALI_URL} (in the output above, this is `kiali-istio-system.127.0.0.1.nip.io`).  You should see the Kiali console login screen.
-+
-image:kiali-login.png[Login Page]
-+
-Log in to the Kiali console using the user name and password that you specified in the custom resource file during installation.
+
+. Log in to the Kiali console using the user name and password that you specified in the custom resource file during installation.
 
 [[kiali-overview-page]]
 == Overview page
 
-After you log in you see the Overview page, which provides you with a quick overview of the health of the various namespaces in your system.
+When you first log in you see the Overview page, which provides a quick overview of the health of the various namespaces that are part of your {ProductShortName}.  You can choose to view the health of your applications, workloads, or services.
 
 image:kiali-overview.png[Overview Page]
 
 [[kiali-graph-page]]
 == Graph page
 
-. Click *Graph* in the left navigation.  The Graph page shows a graph with all the microservices, which are connected by the requests going through them. On this page, you can see how the services interact with each other.
+The Graph page shows a graph of microservices, which are connected by the requests going through them. On this page, you can see how applications, workloads, or services interact with each other.
+
+. Click *Graph* in the left navigation.
 +
-image:kiali-graph.png[kiali graph]
+image:kiali-graph.png[Kiali graph]
 +
-. From the *Namespace* menu, select `bookinfo`.  Now the graph displays only the services in the Bookinfo application.
-. Click *Legend* in the lower left corner.  Kiali displays a window that contains the graph legend.
+. If necessary, select `bookinfo` from the *Namespace* menu.  The graph displays the applications in the Bookinfo application.
+. Click the question mark (?) under the *Namespace* menu to take the Graph Help Tour.
+. Click *Done* to close the Help Tour.
+. Click *Legend* in the lower left corner.  Kiali displays the graph legend.
 +
-image:kiali-legend.png[kiali legend]
+image:kiali-legend.png[Kiali legend]
 +
 . Close the Graph Legend.
 . Hover over the *productpage* node.  Note how the graph highlights only the incoming and outgoing traffic from the node.
-. Click the *productpage* node.  Note how the details on the right side of the page change to show the *productpage* details.
+. Click the *productpage* node.  Note how the details on the right side of the page change to display the *productpage* details.
+
+[[kiali-applications-page]]
+== Applications page
+The Applications page lets you search for and view applications, their health, and other details.
```

Comment:
I'm not sure if what user prefers.
To me, I'd like to have more info about "other details".
For example, "services and workloads that belong to an specific application and access to detailed metrics collected per application". 
(Just to refer to the other tabs user can see from App details page).